src/main/java/org/jcodec/common/model/Picture.java

```diff
@@ -1,17 +1,19 @@
 package org.jcodec.common.model;
+
 import static java.lang.System.arraycopy;
 import static org.jcodec.common.model.ColorSpace.MAX_PLANES;
 
 import java.lang.IllegalArgumentException;
+import java.util.Arrays;
 
 /**
  * This class is part of JCodec ( www.jcodec.org ) This software is distributed
  * under FreeBSD License
  * 
- * A YUV picture
+ * The data is -128 shifted, so 0 is represented by -128 and 255 is represented
```

Comment:
The range -128 to 127 was chosen because it is a native range for the byte type. Alternatively standard range of 0..256 could be used but this creates a necessity to use &0xff all the time in the code. All the transforms that used to be called *8Bit support this range in a form of adjusted clipping ranges. All the arithmetics is the same regardless of the actual range. The range conversion happens only when the final platform-specific image gets extracted (i.e. BufferedImage or Bitmap) if ever.